components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -11,105 +11,121 @@ toc = true
 +++
 
 This guide shows you how to upgrade Chef Automate to IAM v2, perform important administrative operations, and revert back to IAM v1.
-After upgrading to IAM v2, you'll add members to Chef-managed v2 policies, delete a legacy policy, and write a Team Admin v2 policy that lets a Team Admin manage their users and teams.
+After upgrading to IAM v2, you will add members to Chef-managed v2 policies, delete a legacy policy, and write a Team Admin v2 policy that lets a Team Admin manage their users and teams.
 
 ## Upgrade to IAM v2
 
 {{< info >}}
 To get the best possible IAM v2 experience, Chef Automate should be running the latest version before upgrading to IAM v2.
 {{< /info >}}
 
-Perform the upgrade to IAM v2 with `chef-automate iam upgrade-to-v2`; the response should look something like:
+Perform the upgrade to IAM v2 with `chef-automate iam upgrade-to-v2`. The response from your terminal should look something like:
```

Comment:
should we be consistent with using `IAM v2.1` everywhere?

Reply:
i was hoping we could leave most references in the docs to `IAM v2` as is.. im not sure most people are going to have context for 2.0 vs 2.1

Reply:
yeah they probably won't even notice 😅